### ISSUE TITLE
Hyperparameter sweep

### ISSUE THREAD

**Issue by alec-hoyland** [alec-hoyland/TinnitusStimulusFitter.jl#2]
Write a hyperparameter sweep over learning rate and the L1 regularization parameter.
